scrapy/extensions/feedexport.py

```diff
@@ -306,6 +357,16 @@ def _exporter_supported(self, format):
             return True
         logger.error("Unknown feed format: %(format)s", {'format': format})
 
+    def _settings_are_valid(self, uri):
+        """
+        If FEED_STORAGE_BATCH_ITEM_COUNT setting is specified uri has to contain %(batch_time)s or %(batch_id)s
+        to distinguish different files of partial output
+        """
+        if not self.storage_batch_item_count or '%(batch_time)s' in uri or '%(batch_id)s' in uri:
+            return True
+        logger.warning('%(batch_time)s or %(batch_id)s must be in uri if FEED_STORAGE_BATCH_ITEM_COUNT setting is specified')
```

Comment:
```suggestion
        logger.error('%(batch_time)s or %(batch_id)s must be in uri if FEED_STORAGE_BATCH_ITEM_COUNT setting is specified')
```

`_exporter_supported` and `_storage_supported` both log it as an error, and since the result if the feed export being disabled, I guess it makes sense.